reflection/serverreflection_test.go

```diff
@@ -21,6 +21,10 @@ package reflection
 import (
 	"context"
 	"fmt"
+	"github.com/jhump/protoreflect/desc/protoparse"
```

Comment:
Is there an easy way to read the file without depending on another package?
Or check in the `.pb.go` file and use the generated file descriptor?
We should try to minimize third party packages.